src/locales/pt.json

```diff
@@ -0,0 +1,217 @@
+{
+  "Blockchain": {
+    "Mnemonic": "Mnemônico",
+    "Address": "Endereço",
+    "Wallet": "Carteira",
+    "Wallets": "Carteiras",
+    "Coins": "Moedas",
+    "Tokens": "Tokens",
+    "Balance": "Saldo",
+    "Tx hash": "Hash Tx",
+    "Gas": "Gas",
+    "Tax": "Imposto",
+    "Fee": "Taxa",
+    "Memo": "Memo",
+    "decimals": "decimais",
+    "Buy": "Comprar",
+    "Exchanges": "Exchanges",
+    "Fiat": "Fiat",
+    "APY": "APY"
+  },
+  "Form": {
+    "Amount": "Quantidade",
+    "Recipient": "Destinatário",
+    "Name": "Nome",
+    "optional": "opcional",
+    "Submit": "Enviar",
+    "Confirm": "Confirmar",
+    "Reset": "Reset"
+  },
+  "Validate": {
+    "{{label}} is required": "{{label}} é obrigatório",
```

Comment:
Because we can have `obrigatório` and `obrigatória`, I would also suggest `obrigatório/a` but of course, depending on the context, only one will make sense.

Reply:
What I can do in this case, I could translate to "O campo {{label}} é obrigatório", then it will be always "obrigatório"